test_runner/regress/test_disk_usage_eviction.py

```diff
@@ -794,6 +794,16 @@ def less_than_max_usage_pct():
 
     wait_until(2, 2, less_than_max_usage_pct)
 
+    # Disk usage candidate collection only takes into account active tenants.
+    # However, the statvfs call takes into account the entire tenants directory,
+    # which includes tenants which haven't become active yet.
+    #
+    # After re-starting the pageserver, disk usage eviction may kick in *before*
+    # both tenants have become active. Hence, the logic will try to satisfy the
+    # disk usage requirements by evicting everything belonging to the active tenant,
+    # and hence violating the tenant minimum resident size.
+    env.neon_env.pageserver.allowed_errors.append(".*" + GLOBAL_LRU_LOG_LINE)
```

Comment:
Shouldn't this be in the `EvictionEnv.pageserver_start_with_disk_usage_eviction`?

Reply:
I chose the smaller hammer. We can move if the rest of these tests are flaky too.